Tests/Shared.Specs/ExceptionAssertionSpecs.cs

```diff
@@ -889,12 +888,139 @@ public void When_no_exception_should_be_thrown_and_none_was_it_should_not_throw(
             // Arrange
             //-----------------------------------------------------------------------------------------------------------
             var foo = A.Fake<IFoo>();
-
             //-----------------------------------------------------------------------------------------------------------
             // Act / Assert
             //-----------------------------------------------------------------------------------------------------------
             foo.Invoking(f => f.Do()).Should().NotThrow();
         }
+
+        #pragma warning disable CS1998
+        [Fact]
+        public void NotThrowAfter_when_subject_is_async_it_should_not_throw()
```

Comment:
The method name says that it _should not throw_, but the code asserts that `action` _does throw_